test/trackler/implementation_test.rb

```diff
@@ -140,6 +140,17 @@ def test_readme_uses_track_hint_instead_of_setup
     assert_match /This is the content of the track hints file/, implementation.readme
   end
 
+  def test_blurb_not_repeated_if_same_as_start_of_description
+    mock_track = OpenStruct.new(dir: Pathname.new('dont care'), hints: 'dont care')
+    mock_problem = OpenStruct.new(
+      blurb: 'blurb', description: 'blurb then description',
+      name: 'dont-care', slug: 'dont_care', source_markdown: 'dont care'
```

Comment:
It bugs me a bit that the slug has an underscore rather than a hyphen, but I'm not sure its important enough to change, since it's a dummy value that is not used.